### PR TITLE
Leson second dev

### PR DESCRIPTION
**Pull request by alexk2111**

### CODE REVIEW

File: app/src/main/java/com/onix/okucherenko/loginapplication/ui/Page2/Page2Fragment.kt [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+        binding.buttonPage2toPage3.setOnClickListener {
+            onClickButtonPage2toPage3()
+        }
+
+
+    }
+
+    private fun onClickButtonPage2toPage3() {
+
+        if (!binding.radioButtonAnswer1.isChecked &&
```
Comment (shadwork) [inline app/src/main/java/com/onix/okucherenko/loginapplication/ui/Page2/Page2Fragment.kt:68]: rewrite this without !

File: app/src/main/java/com/onix/okucherenko/loginapplication/ui/Page2/Page2Fragment.kt [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+            !binding.radioButtonAnswer2.isChecked &&
+            !binding.radioButtonAnswer3.isChecked &&
+            !binding.radioButtonAnswer4.isChecked &&
+            !binding.radioButtonAnswer5.isChecked
+        ) {
+            val snackBar = Snackbar
+                .make(binding.root, "Make a choice!!!", Snackbar.LENGTH_SHORT)
```
Comment (shadwork) [inline app/src/main/java/com/onix/okucherenko/loginapplication/ui/Page2/Page2Fragment.kt]: move text into android resources

File: app/src/main/java/com/onix/okucherenko/loginapplication/ui/Page3/Page3Fragment.kt [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+
+        viewModelFactory =
+            Page3ViewModelFactory(Page3FragmentArgs.fromBundle(requireArguments()).modelQuiz)
+        viewModel = ViewModelProvider(this, viewModelFactory)[Page3ViewModel::class.java]
+
+        actualPage = viewModel.actualPage
+
+        binding.textViewContext.text = viewModel.quiz.page[actualPage].question[0].content
+
+        binding.checkBoxAnswer1.text = viewModel.quiz.page[actualPage].question[0].answers[0].content
```
Comment (shadwork) [inline app/src/main/java/com/onix/okucherenko/loginapplication/ui/Page3/Page3Fragment.kt]: Rewrite this with loop, for example with forEach

**Review by shadwork — CHANGES_REQUESTED** [review]
неплохо

File: app/src/main/java/com/onix/okucherenko/loginapplication/ui/Page2/Page2Fragment.kt (hunk 2 above)
Comment (alexk2111) [reply app/src/main/java/com/onix/okucherenko/loginapplication/ui/Page2/Page2Fragment.kt]: Done

File: app/src/main/java/com/onix/okucherenko/loginapplication/ui/Page2/Page2Fragment.kt (hunk 1 above)
Comment (alexk2111) [reply app/src/main/java/com/onix/okucherenko/loginapplication/ui/Page2/Page2Fragment.kt:68]: Done

File: app/src/main/java/com/onix/okucherenko/loginapplication/ui/Page3/Page3Fragment.kt (hunk 3 above)
Comment (alexk2111) [reply app/src/main/java/com/onix/okucherenko/loginapplication/ui/Page3/Page3Fragment.kt]: Done. And even more.

**Pull request merged**